### ISSUE TITLE
Enabling external UHD monitor fails

### ISSUE THREAD

**Issue by Fluepke** [swaywm/sway#5235]
> **System information**
> * `swaymsg -t get_version`: sway version 1.4
> * `uname -r`: 5.6.4-arch1-1
> * `cat /proc/cpuinfo | grep 'model name' | head -n1` : model name	: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
> * `swaymsg -t get_outputs`
>     
>     ```
>     [...]
>     Output DP-1 'Philips Consumer Electronics Company PHL BDM4065 0x000002A2' (inactive)
>        Available modes:
>      [...]
>         3840x2160 @ 30.000000 Hz
>         3840x2160 @ 59.997002 Hz
>     ```

Hello,

enabling an external 4K monitor (`output DP-1`) does not work. The display stays inactive.
Enabling it at Full HD does work.
I have tested *3840x2160@60Hz* to be working using i3 with X.

These are the commands used to produce some verbose output:
```bash
echo 0xFE | sudo tee /sys/module/drm/parameters/debug
sudo dmesg -C
dmesg -w >dmesg.log &
sway -d >sway.log 2>&1
```

Here are the log files:
* [dmesg.log](https://gist.github.com/Fluepke/79b8158deccebd3ed1864d61fd432ed5#file-dmesg-log)
* [sway.log](https://gist.github.com/Fluepke/79b8158deccebd3ed1864d61fd432ed5#file-sway-log) 

I find sway.log:893 (and following) noteworthy:
[`[backend/drm/atomic.c:56] DP-1: Atomic commit failed (modeset): Invalid argument`](https://gist.github.com/Fluepke/79b8158deccebd3ed1864d61fd432ed5#file-sway-log-L893)

Thanks,
Fluepke

**Comment by efouladi** [comment]
What happens if you connect the display before starting sway?

**Comment by Fluepke** [comment]
The display was connected before starting sway and producing the above outputs. Sorry for being unclear about that.

Other Wayland compositiors (e.g. Weston) have similar issues, so maybe a bug in a shared library?

**Comment by emersion** [comment]
Duplicate of https://github.com/swaywm/wlroots/issues/1877

```
[ 2207.347436] i915 0000:00:02.0: [drm:skl_allocate_pipe_ddb [i915]] Requested display configuration exceeds system DDB limitations
[ 2207.347452] i915 0000:00:02.0: [drm:skl_allocate_pipe_ddb [i915]] minimum required 402/366
```
